handler/aggregate/pipeline_test.go

```diff
@@ -282,9 +283,24 @@ var _ = Describe("type Sink", func() {
 					Expect(err).ShouldNot(HaveOccurred())
 				})
 
-				It("does not apply historical events when loading", func() {
-					root.ApplyEventFunc = func(dogma.Message, interface{}) {
-						Fail("unexpected call")
+				It("resets the root state", func() {
+					handler.HandleCommandFunc = func(
+						s dogma.AggregateCommandScope,
+						_ dogma.Message,
+					) {
+						// Create() should now return true once again.
+						Expect(s.Create()).To(BeTrue())
+
+						// The root itself should also have been reset to as-new.
+						r := s.Root().(*AggregateRoot)
+						Expect(r.Value).To(Equal(
+							&[]dogma.Message{},
+						))
+
+						// We need to record an event whenever we create to
+						// prevent a panic, we do this after our assertion that
+						// the root is empty otherwise we this event applied.
```

Comment:
I'm not sure what you were aiming for here:
> otherwise we this event

The sentence could probably do with a full stop above also? "prevent a panic, we do this"